src/core/utils/fs.cpp

```diff
@@ -453,6 +453,18 @@ QString Utils::Fs::QDesktopServicesDownloadLocation()
 #endif
 #endif
 
+#if defined(Q_OS_WIN)
+#if (QT_VERSION < QT_VERSION_CHECK(5, 0, 0))
+    if (QSysInfo::windowsVersion() <= QSysInfo::WV_XP)  // Windows XP
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).absoluteFilePath(
+                    QCoreApplication::translate("fsutils", "Downloads"));
+    else
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::HomeLocation)).absoluteFilePath("Downloads");
+#else
+    return QStandardPaths::writableLocation(QStandardPaths::DownloadLocation);
```

Comment:
Qt5 does use XDG_DOWNLOAD_DIR for QStandardPaths::DownloadLocation
Also in Windows since Vista (no idea about XP) folder's aren't translated on filesystem, but it's done by Explorer. So on disk you've "Downloads" not translated, but if you look with Explorer you'll see it translated while actually on disk it's not.